lxqt-base/lxqt-admin/lxqt-admin-0.17.0.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="LXQt system administration tool"
+HOMEPAGE="https://lxqt.github.io/"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
+fi
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+
+BDEPEND=">=dev-util/lxqt-build-tools-0.9.0"
+DEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	=lxqt-base/liblxqt-$(ver_cut 1-2)*
+	kde-frameworks/kwindowsystem:5
+	sys-auth/polkit-qt[qt5(+)]
```

Comment:
Optional: Drop `[qt5(+)]` by depending on `>=sys-auth/polkit-qt-0.113.0`, our first version to have dropped Qt4.